src/app/stores/presence-status-store.ts

```diff
@@ -51,45 +51,45 @@ export class PresenceStatus {
     const os = isWindowsOS ? 'windows' : isMac ? 'macOS' : 'linux';
     const theme = nativeTheme.shouldUseDarkColors ? 'light' : 'dark';
     const assetsPath = `src/renderer/assets/presence-status/${os}/${theme}`;
-
+    let fileExtension = 'png';
+    let iconPlace = '';
+    switch (place) {
+      case 'tray':
+        iconPlace = '-tray';
+        fileExtension = isWindowsOS ? 'ico' : isMac ? 'png' : 'png';
```

Comment:
So now we will have to use ICO Format? Hmm I read it Fabio said he couldn't provide the ICO ones?!! Interesting.
Just a compliment comment